publiccode.yml

```
@@ -0,0 +1,99 @@
+publiccodeYmlVersion: 0.4.0
+name: Piattaforma Digitale Nazione Dati - Interoperabilità
+applicationSuite: PDND
+url: https://github.com/pagopa/interop-be-monorepo
+landingURL: https://www.interop.pagopa.it/
+releaseDate: 2025-08-07
```

is this the release date of the platform?
In that case, may we change it to this?
```suggestion
releaseDate: 2022-10-01
```